### DISCUSSION TITLE
New project proposal: fxgl-animations

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1292] [Show and tell]
Hi:

I have been working on my own project here:
https://github.com/chengenzhao/fxcity
and have re-writed animation part of FXGL
using Java code and also have extra functionalities
and is it a good idea to have a new sub project of FXGL 
we could name it fxgl-animation
which provides extra functions of animation
like animated svg, animated 3d model, animated canvas etc.?

Thanks

**Comment by AlmasB** [comment]
Thanks for this. I have a student who will start working on this from October. They are particularly interested in designing and developing the 3d animation aspects. Once they put together a proposal, we can revisit this and see how to proceed forward.

**Reply by chengenzhao** [reply]
Cool
For 3D animation we probably need PixelBuffer and project panama to enhance the native rendering like Shader in other languages etc.
